sonar-db/src/test/java/org/sonar/db/version/IntegerColumnDefTest.java

```diff
@@ -76,6 +76,11 @@ public void builder_setColumnName_sets_name_field_of_IntegerColumnDef() {
   }
 
   @Test
+  public void default_value_is_not_supported() {
+    assertThat(newIntegerColumnDefBuilder().setColumnName("a").build().getDefaultValue()).isNull();
```

Comment:
given the name of the test, I thought a call to some setDefaultValue method was missing
actually, the tests ensures that getDefaultValue returns null